### DISCUSSION TITLE
Installation error

### DISCUSSION THREAD

**Discussion by lensofq** [ANTsX/ANTs#1412] [Q&A]
Hello, I am trying to build ANTs on Mac , following the steps on GitHub instructions. 
During the build step 
make 2>&1 | tee build.log

I am encountering this error: 
CMake Error at Examples/cmake_install.cmake:41 (file):
  file cannot create directory: /opt/ANTs/lib.  Maybe need administrative
  privileges.
Call Stack (most recent call first):
  cmake_install.cmake:42 (include)

make: *** [install] Error 1

How do I resolve this error?

**Comment by cookpa** [comment]
This can be resolved by setting `CMAKE_INSTALL_PREFIX` to somewhere you can write.
